packages/host/app/components/operator-mode/card-schema-editor.gts

```diff
@@ -136,7 +137,10 @@ export default class CardSchemaEditor extends Component<Signature> {
       .right {
         display: flex;
         align-items: center;
-        gap: var(--boxel-sp-xxs);
+      }
+
+      .right > *:not(:nth-last-child(3)) {
```

Comment:
Why 3? Looks oddly specific - will this break when new elements are added?

Reply:
I removed it and add the margin to a specific class. [d398580](https://github.com/cardstack/boxel/pull/722/commits/d398580b8f7b9a7843782a9ee4933a43667c6c03)